docs/azure-resources.md

```diff
@@ -0,0 +1,86 @@
+# THIEF Azure resources
+
+verion .1
+
+## Organization
+
+Azure resources used by THIEF are split into _instance_ and _shared_ resources.
+* Instance resources are the things we're testing, such as IoT Hub and DPS instances.
+* Shared resources are the things we use for reporting and coordination of tests.
+
+This division is necessary because we want test isolation (instance resources) and also a global view of tests (shared resources).
+* Test isolation means that one set of tests (say Python) cannot cause another set of tests (say .net) to fail by exhausing things like Hub resources.  For example, if a Python test causes C2D throttling, we don't want this to affect C# tests.
+* A global view of tests means that we can have one dashboard showing all test runs with a single way to find all test results.
+
+Instance and shared resources may be in different Azure subscriptions.
+This is necessary to support developers running under their own subscriptions while still reporting to the global stores.
+
+## Shared THIEF resources
+
+All of these resources are under the subscription `THIEF_SHARED_SUBSCRIPTION_ID` in the `THIEF_SHARED_RESOURCE_GROUP` resource group.
+
+`THIEF_SHARED_KEYVAULT_NAME` has a keyvault with secrets for all shared resources.
+This kevault is used to seed the instance keyvault and many of the secrets in this keyvault are duplicated in the instance keyvault
+
+`THIEF_APP_INSIGHTS_CONNECTION_STRING` is the entrypoint for the shared App Insights intance.
```

Comment:
In .NET I'm just using the AppInsights Key, not a connection string. What options do you have in Python?

Reply:
I think it's the same thing -- `InstrumentationKey=<guid>`.  The Python `AzureEventHandler` class, etc, call the _parameter_ `connection_string`, so I carried that forward.

Reply:
So not a connection string??

Reply:
Looking at the AI portal, it appears to be a partial connection string (InstrumentationKey only, IngestionEndpoint not included).  It sounds like you're saying that a raw Instrumentation Key would be more useful.  Is that right?  If so, I have no problem changing it to a key and renaming it.

Reply:
The raw AI key is all I need for .NET, yes.

Reply:
fixed in #48